app/controllers/RestApi.scala

```diff
@@ -328,5 +329,7 @@ class RestApi @Inject() (
     accessRightDao.delete(id).map(count => if (count == 0) NotFound else Ok)
   }}
 
-
+  def createPool1User() = Action.async(validateJson[Pool1User]) { request => 
```

Comment:
Do we need a possibility to create Pool 1 user by a webservice? I thought they will be imported during migration...
(and if we need it, we have secure it...)